assets/app/components/linkButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const propTypes = {
+  className: React.PropTypes.string,
+  text: React.PropTypes.string,
+  href: React.PropTypes.string
+}
+
+const LinkButton = ({className, text, href}) =>
```

Comment:
Woah, you can make components with just function like this?

What does `React` do in this case?